### PR TITLE
[Feature] [k8s] Add k8s CI test and GitHub Actions

### PR DESCRIPTION
**Pull request by Xuedixuedi**
## Purpose of the pull request

Add K8S CI in github actions.

## Brief change log

Add a k8s test to verify follow points:

- Lint test
- Chart Install test: mainly for dolphinscheduler chart and zookeeper, postgresql
- Check server status ready

Files changed:
- .github/workflows/ci_k8s.yml  CI to Github Actions
- docker/kubernetes/check-node.sh check k8s node already
- docker/kubernetes/check.sh check server 

## Verify this pull request

This change added tests and can be verified as follows:

- Added dolphinscheduler k8s test to ensure we can build a correct and healthy cluster on k8s.

### CODE REVIEW

**Comment by kezhenxu94** [comment]
@Xuedixuedi you use so many third party GitHub Actions that are not allowed in Apache infra, please try to use submodules

**Comment by CalvinKirs** [comment]
@kezhenxu94  hi PTAL,thx

File: .github/workflows/ci_k8s.yml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+env:
+  DOCKER_DIR: ./docker
+  LOG_DIR: /tmp/dolphinscheduler
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest # runs-on 是必填字段
```
Comment (kezhenxu94) [inline .github/workflows/ci_k8s.yml]:
```suggestion
    runs-on: ubuntu-latest
```

File: .github/workflows/ci_k8s.yml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Check License Header
+        uses: apache/skywalking-eyes@ec88b7d850018c8983f87729ea88549e100c5c82
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+
+      - name: Set up Python
```
Comment (kezhenxu94) [inline .github/workflows/ci_k8s.yml:49]: Why we need Python?

File: .github/workflows/ci_k8s.yml [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Fetch history
```
Comment (kezhenxu94) [inline .github/workflows/ci_k8s.yml:54]: Why we need the git history?

**Review by kezhenxu94 — CHANGES_REQUESTED** [review]
> @Xuedixuedi you use so many third party GitHub Actions that are not allowed in Apache infra, please try to use submodules

This is not even addressed yet, the new workflow is not even executed.

**Pull request closed without merging**